### DISCUSSION TITLE
FAQ mentioned only 1 file in the compressed file will be scanned. Which format is supported?

### DISCUSSION THREAD

**Discussion by kid1412621** [Swordfish90/Lemuroid#669] [Q&A]
Only support zip file supported to be scanned. How about the format in the zip file being scanned?

**Comment by cyruzwings** [comment]
> Only support zip file supported to be scanned. How about the format in the zip file being scanned?

do you mean what extension supported?
then that depends by game console you want to play
for example gameboy advance, it support **_.gba_** (.gba that compress to zip also works)

**Reply by kid1412621** [reply]
How about 7z or other compressed file formats?
